src/utils.ts

```diff
@@ -5,6 +5,8 @@ const splitLines = require('split-lines');
 import * as fs from 'fs';
 const dotf = require('dotf');
 const read = require('read-file');
+import { Spinner } from 'cli-spinner';
+import { ERROR } from '../index.js';
```

Comment:
I'm not a huge fan of importing things into Utils from the main script. I'd rather have `ERROR` in `src/utils.ts` or in `src/commands` and import from there if needed.